plugin/db/pg/pg.go

```diff
@@ -357,41 +357,44 @@ func (driver *Driver) getUserList(ctx context.Context) ([]*db.User, error) {
 }
 
 // Execute executes a SQL statement.
-func (driver *Driver) Execute(ctx context.Context, statement string, useTransaction bool) error {
+func (driver *Driver) Execute(ctx context.Context, statement string) error {
 	// We don't use transaction for creating databases in Postgres.
 	// https://github.com/bytebase/bytebase/issues/202
-	if !useTransaction {
-		f := func(stmt string) error {
-			// For the case of `\connect "dbname";`, we need to use GetDbConnection() instead of executing the statement.
-			if strings.HasPrefix(stmt, "\\connect ") {
-				parts := strings.Split(stmt, `"`)
-				if len(parts) != 3 {
-					return fmt.Errorf("invalid statement %q", stmt)
-				}
-				_, err := driver.GetDbConnection(ctx, parts[1])
-				return err
-			}
+	var remainingStmts []string
+	f := func(stmt string) error {
+		// This is a fake CREATA DATABASE statement. Engine driver will recognize it and establish a connect to create the database.
```

Comment:
```suggestion
		// This is a fake CREATE DATABASE statement. Engine driver will recognize it and establish a connection to create the database.
```

Reply:
My feeling is this comment should be put on the caller instead of here.

Or we should explain how it's expected to be used instead. IIUC, there is a trick here which allows us to get rid of useTransaction bool

Reply:
Done, added to method interface.